### PR TITLE
added vex-js dialog box

### PR DESCRIPTION
**Pull request by harshitks2203**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -17,6 +17,9 @@
   <link rel="stylesheet" href="css/simplescrollbars.css">
   <link rel="stylesheet" href="css/base16-tomorrow-light.css">
   <link rel="stylesheet" href="css/base16-ocean-dark.css">
+  <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/vex-js/3.0.0/css/vex.min.css" /> -->
```
Comment (apollonian11) [inline app/index.html]: We don't need a CDN now that we're using styles from the package itself (line 21, and 22), right?

Comment (apollonian11) [reply app/index.html]: Oh, apologies, it's just a comment. 👍

File: app/taskMenu.js [hunk 2]
```diff
@@ -21,7 +21,29 @@ function setAutoRun() {
 }
 
 function getUserConnect(callback) {
-  document.getElementById("connect").addEventListener("click", () => { getIP(callback) });
+  document.getElementById("connect").addEventListener("click", () => { showDialog(callback) });
+}
+
+function showDialog(callback) {
```
Comment (apollonian11) [inline app/taskMenu.js]: We're not validating the text input here @harshitks2203. Users might not enter a correct URL.

**Comment by Jay9596** [comment]
This would __not__ work as work as intended. 

There are some inconveniences that need to be addressed.
- Remove the unnecessary comment in html file. 
- Why remake a function to do the same job as `getIP`. You could've just added the functionality in place of the comments and called it a day.
- The `getIP` function is called by wrong "data". Instead of a callback function, you're giving it a IP string as argument.
- Some basic IP checks are needed in place. User can type anything, _notIP_ and expect some feedback.
(Before calling callback, just check and 'alert' the user.)

File: app/taskMenu.js (hunk 2 above)
Comment (harshitks2203) [reply app/taskMenu.js]: ok i will add validation too.

**Pull request closed without merging**